test/integration/suites/upgrade/versions.txt

```diff
@@ -5,3 +5,5 @@
 1.4.0
 1.4.1
 1.4.2
+1.4.3
```

Comment:
1.3.x versions should be removed here.

Reply:
That doesn't happen until after we release 1.5.0 and add it here, right?

Reply:
Now I need to look to see what we've done in the past :rofl:

Reply:
Ok, I re-read our comment in the upgrade IT README and you're right